custom_components/hacs/update.py

```diff
@@ -148,9 +148,20 @@ async def async_release_notes(self) -> str | None:
                 self.repository.data.last_version = next(iter(self.repository.data.published_tags))
 
         release_notes = ""
-        if len(self.repository.releases.objects) > 0:
-            release = self.repository.releases.objects[0]
-            release_notes += release.body
+        above_current_version = False
```

Comment:
Before this loop, make sure the current version is in the list.
If not, we can assume it is really old and only show the current for now (will replace that with a diff link later anyway).